modules/oke/variables.tf

```diff
@@ -57,6 +57,12 @@ variable "enable_pv_encryption_in_transit" {
   type = bool
 }
 
+variable "create_policies" {
```

Comment:
In the submodule, we don't currently set description and defaults because we are treating as an internal module. Let's keep the consistency for now.